client/pubspec.yaml

```diff
@@ -39,11 +39,11 @@ dependencies:
   client_common:
     git:
       url: git@github.com:lenra-io/client-common.git
-      ref: v1.0.0-beta.40
+      ref: v1.0.0-beta.41
   lenra_ui_runner:
     git:
       url: git@github.com:lenra-io/lenra_ui_runner.git
-      ref: v1.0.0-alpha.5
+      ref: v1.0.0-alpha.1
```

Comment:
Why did you downgrade version ?

Reply:
There was a problem with semantic release on lenra_ui runner ^^ We had to remove all the alpha tags and start from zero.